criu/include/log.h

```diff
@@ -60,6 +60,8 @@ void flush_early_log_buffer(int fd);
 
 #define pr_perror(fmt, ...) pr_err(fmt ": %s\n", ##__VA_ARGS__, strerror(errno))
 
+#define pr_pwarn(fmt, ...) pr_warn(fmt ": %s\n", ##__VA_ARGS__, strerror(errno))
```

Comment:
As you add `pr_pwarn` new printing function, you also need to update `lint` accordingly.

Reply:
Done.